rcjaRegistration/events/views.py

```diff
@@ -400,3 +401,258 @@ def summaryReport(request):
+                division_dict['results'][event_no] = {'teachers':division['teachers'],'students':division['students']}
+                division_cat_dict['teachers'][event_no] += division['teachers']
+                division_cat_dict['students'][event_no] += division['students']
+                division_cat_dict['divisions'][division['name']] = division_dict
+            divisions['categories'][division_cat['name']] = division_cat_dict
+            divisions['students'][event_no] += division_cat_dict['students'][event_no]
+            divisions['teachers'][event_no] += division_cat_dict['teachers'][event_no]
+
+
+    for division_cat in divisions['categories']:
+        rows = len(divisions['categories'][division_cat]['divisions'].keys())
+        divisions['categories'][division_cat]['rows']=rows+1
+    
+    # Schools
+    schools = {'teachers':[0]*workshop_number,'students':[0]*workshop_number,'schools':{}}
+    for event_no, event in enumerate(events):
+        for school in event['schools']:
+            school_dict = schools['schools'].get(school['name'], {'name':school['name'],'teachers':[0]*workshop_number,'students':[0]*workshop_number})
+            school_dict['teachers'][event_no] += school['teachers']
+            school_dict['students'][event_no] += school['students']
+            schools['schools'][school['name']] = school_dict
+            schools['students'][event_no] += school_dict['students'][event_no]
+            schools['teachers'][event_no] += school_dict['teachers'][event_no]
+
+    context = {'divisions':divisions,
+               'schools':schools,
+               'events':events,}
+    return render(request, 'events/adminMultiWorkDetails.html', context)
+
+def getAdminCompSummary(event):
+    # Divisions
+    divisionList = event.divisions.values()
+    division_categories = {}
+    division_teams = 0
+    division_students = 0
+    for division in divisionList.all():
+        teams = Team.objects.filter(event = event).filter(division = division["id"])
+        students = 0
+        for team in teams:
+            students += Student.objects.filter(team=team).count()  
+        division_students += students
+        division_teams += teams.count()
+        divisionDict = {
+            'name': division["name"],
+            'teams': teams.count(),
+            'students': students,
+        }
+
+        if division['category_id'] in division_categories:
+           division_categories[division['category_id']]['divisions'].append(divisionDict)
+           division_categories[division['category_id']]['rows'] += 1
+           division_categories[division['category_id']]['students'] += students
+           division_categories[division['category_id']]['teams'] += teams.count()
+        else:
+            division_categories[division['category_id']]={'name':DivisionCategory.objects.get(id=division['category_id']).name,
```

Comment:
@Harr0101 this throws a DoesNotExist: DivisionCategory matching query does not exist. error if the division category for the division is blank.